noir-projects/noir-protocol-circuits/crates/rollup-lib/src/block_root/block_root_rollup_inputs.nr

```diff
@@ -93,6 +95,24 @@ impl BlockRootRollupInputs {
 
         let total_fees = components::accumulate_fees(left, right);
 
+        // unsafe {
```

Comment:
Leftovers?

Reply:
Not really, I tend to leave these logs behind so I can easily re-enable them if I need to debug anything. We need to wire LOG_LEVEL into noir as well I guess.